PWR/PWR_RTC/Core/Src/main.c

```diff
@@ -64,18 +64,19 @@ int main(void)
 	RTC->WPR = RTC_KEY1;
 	RTC->WPR = RTC_KEY2;
 	// --- RTC's unlocked
-	RTC->CR &= ~RTC_CR_WUTE;					//disables the wakeup timer
-	while(!(RTC->ISR & RTC_ISR_WUTWF));			//polling WUTWF until it is set
-	RTC->ISR &= ~RTC_ISR_WUTF;					//clears RTC wakeup flag
-	RTC->ISR |= RTC_ISR_INIT;					//RTC enters initialization mode
-	while(!(RTC->ISR & RTC_ISR_INITF));			//polling initialization mode flag
-	RTC->PRER = (36 << RTC_PRER_PREDIV_A_Pos);	//set f_apre to 1 kHz
-	RTC->WUTR = RTC_30_SECONDS;					//wakeup timer set to 30 seconds
-	RTC->CR	 |=	RTC_CR_WUCKSEL_2	|			//10x: ck_spre (usually 1 Hz) clock is selected
-				RTC_CR_WUTIE;					//enables periodic wakeup interrupt (to exit from standy mode)
-	RTC->ISR &= ~RTC_ISR_INIT;					//RTC exits initialization mode
-	while(RTC->ISR & RTC_ISR_INITF);			//polling initialization mode flag
-	PWR->CR &= ~PWR_CR_DBP;						//disable write access to the RTC registers
+	RTC->CR &= ~RTC_CR_WUTE;						//disables the wakeup timer
+	while(!(RTC->ISR & RTC_ISR_WUTWF));				//polling WUTWF until it is set
+	RTC->ISR &= ~RTC_ISR_WUTF;						//clears RTC wakeup flag
+	RTC->ISR |= RTC_ISR_INIT;						//RTC enters initialization mode
+	while(!(RTC->ISR & RTC_ISR_INITF));				//polling initialization mode flag
+	RTC->PRER = (( 36 << RTC_PRER_PREDIV_A_Pos) |	//set f_apre to 100 Hz

```

Comment:
It's 1kHz right?

Reply:
no, I changed it. I would comment when I finished reading about the clock.

I've set PREDIV_A to 36 and PREDIV_S to 999, so I get a 1 Hz input for the RTC

Reply:
Yes, but in PREDIV_A you have 1kHz, not 100Hz

Reply:
right... I calculated first to be 100 Hz, but PREDIV_A would be to high

Reply:
Fix this and request the review again, please.